_docs/install/prerequisites/clusterdiscovery.md

```diff
@@ -36,6 +36,8 @@ Replicas are unavailable in a single node install.
 The environment variable `JOIN` is used to pass clustering information to the StorageOS node.
 This environment variable can contain two types of information, a cluster token or a set of IP addresses.
 
+To ensure that the cluster was successfully created, it is wise to [check the cluster status]({%link _docs/install/health.md %}) after creating the StorageOS instances.
```

Comment:
I would reword this line to:

You can [check the cluster status](...) to confirm a successful installation.